.github/workflows/ci-e2e.yml

```diff
@@ -191,6 +191,7 @@ jobs:
                   GITHUB_ACTION_RUN_URL="${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
                   CELERY_METRICS_PORT=8999
                   CLOUD_DEPLOYMENT=E2E
+                  ENCRYPTION_SALT_KEYS=9bf7f7c13cf2148a23d6b07557f95cb7
```

Comment:
ProTip™️: `deadbeef` is legit hex, as is `00beef0000beef0000beef0000beef00` if you want something that looks less like a [real key](https://github.com/PostHog/posthog/pull/25022#discussion_r1764642046)

Reply:
nice